core/src/text/fontContext.cpp

```diff
@@ -2,8 +2,13 @@
 #define FONTSTASH_IMPLEMENTATION
 #include "fontstash.h"
 
+#define LOGE(fmt, ...) do { logMsg("FontContext/Error: " fmt "\n", ## __VA_ARGS__); } while(0)
+#define LOGW(fmt, ...) do { logMsg("FontContext/Warn: " fmt "\n", ## __VA_ARGS__); } while(0)
```

Comment:
The comma should only be inserted when there are arguments to be able to use the macro with LOGE("just a message")

The do while wrapping ensures that the macro behaves also syntactically like regular functions:   
http://stackoverflow.com/questions/1067226/c-multi-line-macro-do-while0-vs-scope-block